driver/src/driver/scheduler.rs

```diff
@@ -71,7 +71,7 @@ impl Default for AuctionTimingConfiguration {
 }
 
 /// The different kinds of schedulers.
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
nit: do you still need these here and below?

Reply:
Yes, I believe I do.

Reply:
```suggestion
#[derive(Debug)]
```